Automattic/jetpack
# AI Assistant: Add proofread component

### Depends on #31954 ⚠️
Related with #31946 

## Proposed changes:

* Add new `Proofread` component.
* Load it in `AiAssistantPluginSidebar` component.

### Other information:


## Jetpack product discussion


N/A

## Does this pull request change what data or activity we track or use?


N/A

## Testing instructions:






* Follow the steps to activate the sidebar from #31954 
* Check if the `AI Assistant` panel has the proposed text and button opens an empty `Modal`.
